sylius/partial-shipment-plugin/0.1/manifest.json

```
@@ -0,0 +1,9 @@
+{
+    "aliases": ["partial-shipment"],
```

In such cases, I think we can add more aliases to prevent typos. The first one may be official, but the second one still will install the correct library when the user makes a typo.

```suggestion
    "aliases": ["partial-shipment", "partialshipment"],
```